airbyte-integrations/connectors/source-github/Dockerfile

```
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.4.10
+LABEL io.airbyte.version=0.4.11
```

```suggestion
LABEL io.airbyte.version=0.5.0
```

You're adding a new feature, so a minor bump is better 😄